db/questions.yml

```diff
@@ -93,7 +93,7 @@ kitchen_problem:
     - text: Electrical
       next: kitchen_electrical_problem
 
-    - text: Heating
+    - text: Heating or Hot water
```

Comment:
How about if we split these into two options, so it's easier to scan down the page?

Reply:
That makes perfect sense, happy for that. 

If we can include the 'I have no heating' option within the 'heating' options as well, incase user doesn't pick up on initial emergency filter page